v2/client/client_default.go

```diff
@@ -12,7 +16,17 @@ import (
 // client, all outbound events will have a time and id set if not already
 // present.
 func NewDefault() (Client, error) {
-	p, err := http.New()
+	p, err := http.New(
+		http.WithMiddleware(func(h nethttp.Handler) nethttp.Handler {
+			return &ochttp.Handler{
```

Comment:
We do not want to introduce the opencensus libs in the default client, please apply this change to https://github.com/cloudevents/sdk-go/blob/589ccf58301886a6aa2165ba77f16f4c02cf4260/v2/client/client_observed.go#L14

Reply:
Since this is HTTP specific it probably belongs in the HTTP protocol instead, maybe as a protocol option?

Reply:
then `http.NewObserved()`?